tests/seq/thardalignmentconstraint.nim

```diff
@@ -0,0 +1,17 @@
+discard """
+disabled: true
+"""
+
+# does not yet work
```

Comment:
Well the tests are green now, at least.

Reply:
It is disabled. Maybe I should remove it since you don't like disabled tests.